test/suite/hotspots.test.ts

```diff
@@ -240,6 +240,39 @@ suite('Hotspots Test Suite', async () => {
     expect(scanWasPerformedForFolder.path).to.equal(workspaceFolder2.uri.path);
   });
 
+  test('should not ask for confirmation if less files than threshold', async () => {
+    let confirmationAsked = false;
+    const shouldAnalyse = await filesCountCheck(1, ()=> {
+      confirmationAsked = true;
+      return Promise.resolve('Don\'t analyse');
```

Comment:
With an `enum`, this (and other occurrences) could return the related enum value.